packages/event-bus-redis/src/services/event-bus-redis.ts

```diff
@@ -168,9 +138,12 @@ export default class RedisEventBusService extends AbstractEventBusModuleService
     const subscribersResult = await Promise.all(
       subscribersInCurrentAttempt.map(async ({ id, subscriber }) => {
         return await subscriber(data, eventName)
-          .then((data) => {
+          .then(async (data) => {
             // For every subscriber that completes successfully, add their id to the list of completed subscribers
             completedSubscribersInCurrentAttempt.push(id)
+
+            await (this.moduleOptions_.removeJobStrategy?.(job) ?? this.defaultRemoveJobStrategy(job)).catch(() => void 0)
```

Comment:
suggestion:
I think we should declare above the class
```ts
const defaultRemoveJobStrategy = (job) => {
  setTimeout(async () => await job.remove(), 10000)
}
```

and then, in the constructor assign on the class 

```ts
this.defaultRemoveJobStrategy = this.moduleOptions_.removeJobStrategy ?? defaultRemoveJobStrategy
```

And finally here, `await this.defaultRemoveJobStrategy(job)`

Reply:
I like it!